src/lib/Libpython/pbs_python_svr_internal.c

```diff
@@ -5900,6 +5917,7 @@ _pbs_python_event_to_request(unsigned int hook_event, hook_output_param_t *req_p
 					"Failed to populate request structure!");
 				return -1;
 			}
+			print_svrattrl_list("pbs_populate_svrattrl_from_python_class==>", &((struct rq_queuejob *)(req_params->rq_job))->rq_attr);
 			/* fall through here */
 		case HOOK_EVENT_EXECHOST_PERIODIC:
```

Comment:
There are 2 different things being printed out: req_params->rq_job and req_params->vns_list.